siatest/renter/consts.go

```diff
@@ -0,0 +1,19 @@
+package renter
+
+import (
+	"os"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/siatest"
+)
+
+// renterTestDir creates a temporary testing directory for a renter test. This
+// should only every be called once per test. Otherwise it will delete the
+// directory again.
+func renterTestDir(t *testing.T) string {
```

Comment:
smae deal here with the panic and the passing around a testing.T object instead of a string that's intended to be the result of t.Name